lasagne/layers/merge.py

```diff
@@ -4,6 +4,12 @@
 
 
 __all__ = [
+    "CROP_NONE",
+    "CROP_LOWER",
+    "CROP_CENTER",
+    "CROP_UPPER",
```

Comment:
It just occurred to me that these will end up directly in the `lasagne.layers` namespace now. Is it really necessary to define those constants? In all other places, we just use strings directly (e.g., for the different padding modes in convolutions). Defining global constants doesn't seem very Pythonic to me. Do others have any opinion? (Sorry for noticing this so late, I should have thought about it earlier!)